tldw_Server_API/app/core/Ingestion_Media_Processing/OCR/backends/llamacpp_ocr.py

```
@@ -0,0 +1,568 @@
+        image_path = os.path.join(tmpdir, "page.png")
+        with open(image_path, "wb") as handle:
+            handle.write(image_bytes)
+        command = render_argv_template(
+            profile.argv,
+            model_path=profile.model_path,
+            image_path=image_path,
+            prompt=prompt,
+            host=None,
+            port=None,
+        )
+        completed = subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)